tests/resolvers/AgendaItem/categories.spec.ts

```diff
@@ -43,18 +42,7 @@ beforeAll(async () => {
     creatorId: testUser?._id,
   });
 
```

Comment:
@Atharva-Kanherkar I think we need to modify tests according to the AppUserProfile instead of deletion. It leads to low test coverage

Reply:
Yes, i am trying to do exactly that. Thank you